lib/moonshot/change_set.rb

```diff
@@ -89,11 +92,11 @@ def wait_for_change_set
           return
         end
 
-        if Time.now.to_i > start + 30
-          raise 'ChangeSet did not complete creation within 30 seconds!'
+        if Time.now.to_i > start + wait_seconds
+          raise "ChangeSet did not complete creation within #{wait_seconds} seconds!"
         end
 
-        sleep 0.25 # http://bit.ly/1qY1ZXJ
+        sleep 5 # http://bit.ly/1qY1ZXJ
```

Comment:
This seems a bit much to apply across the board, but I'm not familiar with the issue